src/main/java/uk/gov/ida/verifyserviceprovider/factories/saml/ResponseFactory.java

```diff
@@ -182,7 +182,29 @@ public EidasAssertionTranslator createEidasAssertionService(
                 new SignatureValidatorFactory(),
                 europeanIdentityConfiguration.getAllAcceptableHubConnectorEntityIds(),
                 new UserIdHashFactory(hashingEntityId)
-                );
+        );
+    }
+
+    public EidasUnsignedAssertionTranslator createEidasUnsignedAssertionService(
```

Comment:
I see this is already the convention, but I'm finding it confusing that functions that provide a 'Translator' are named as if they are providing a 'Service'. Is a 'Translator' a service, I guess so, but it seems strange that we flip how we call it. There's also some inconsistency here about how we are using a factory instead of the DI that we do elsewhere, but whatevs :-)

Reply:
Yeah, you're totally right. I'm going to change them to be 'Translators' instead of 'Services'.

As for the DI, we don't use Guice to inject stuff for us in the VSP like we do in hub, msa etc. Is that what you mean, or is it an inconsistency in how we do DI in just the VSP?